pkg/controller/rest/presentproof/models.go

```diff
@@ -140,6 +140,10 @@ type presentProofAcceptPresentationRequest struct { // nolint: unused,deadcode
 	// in: path
 	// required: true
 	PIID string `json:"piid"`
+
+	// in: body
+	// required: true
+	Names []string `json:""`
```

Comment:
missed the name on the json tag

Reply:
It is not.  This is only for the `swagger`. The empty JSON tag means that the request body should be an array.